core/src/main/resources/jenkins/model/Jenkins/_safeRestart.jelly

```diff
@@ -31,7 +31,7 @@ THE SOFTWARE.
 			<j:choose>
 				<j:when test="${app.lifecycle.canRestart()}">
 					<form method="post" action="safeRestart">
-						${%Are you sure you want to restart Jenkins? Jenkins will restart once all running jobs are finished.}
+						${%Are you sure you want to restart Jenkins? Jenkins will restart once all running jobs are finished. (Pipeline builds may prevent Jenkins from restarting for a short period of time in some cases, but if so, they will be paused at the next available opportunity and then resumed after Jenkins restarts.) }
```

Comment:
Oops, yeah parentheses cannot be used inside of `${% }` directly, so the value needs to be moved to a properties file.

Reply:
Also, this broke all of the existing localizations.

Reply:
Actually https://github.com/jenkinsci/jenkins/pull/3871 broke the existing localizations first. This PR only broke that ones that had been fixed in the meantime.

Reply:
That said, I'm not sure if our preference is to intentionally break existing localizations when the English text changes so that localizations are never missing information compared to the latest version of the English text, or to preserve existing localizations in cases where the English text has not changed significantly.

Reply:
See https://github.com/jenkinsci/jenkins/pull/7286.